index.js

```diff
@@ -20,75 +16,123 @@ let defaults = {
 }
 
 exports.register = function(server, options, next) {
-  let settings = _.clone(options);
+  // setup defaults:
+  var settings = _.clone(options);
   settings = _.defaults(settings, defaults);
-  let endpoint = settings.endpoint;
-  let log = settings.log;
-  let blocklist = settings.blocklist;
-  let validator = settings.validator;
+  var endpoint = settings.endpoint;
+  // you can pass your own logging function as options.log
+  // by default we use hapi's default logging and tag it to this plugin,
+  var log = settings.log ? settings.log : function log(message, tags){
```

Comment:
no need to be able to override this.  it's common place for to just use server.log